cmake/patches/ngraph/ngraph_protobuf.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/cmake/external_protobuf.cmake b/cmake/external_protobuf.cmake
+index 47977b3..1a66e1c 100644
+--- a/cmake/external_protobuf.cmake
++++ b/cmake/external_protobuf.cmake
+@@ -23,7 +23,7 @@ include(ExternalProject)
+
+ # This version of PROTOBUF is required by Microsoft ONNX Runtime.
+ set(NGRAPH_PROTOBUF_GIT_REPO_URL "https://github.com/protocolbuffers/protobuf")
+-set(NGRAPH_PROTOBUF_GIT_TAG "v3.5.2")
++set(NGRAPH_PROTOBUF_GIT_TAG "48cb18e5c419ddd23d9badcfe4e9df7bde1979b2")
```

Comment:
which protobuf version is this?

Reply:
It is the version onnxruntime uses, taken from here https://github.com/Microsoft/onnxruntime/blob/07e6dfa/cgmanifest.json#L250

Reply:
looks like that is v3.6.1

Reply:
btw, can you also add the ngraph project to cgmanifest.json (follow mkl-dnn as example)

Reply:
Sure, I'll add it

Reply:
do you still plan on add v3.6.1 instead of commit id? 
please resolve items after they're completed so it's clear.